engine/consensus/matching/core_test.go

```diff
@@ -510,6 +571,7 @@ func (ms *MatchingSuite) TestSealableResultsEmergencySealingMultipleCandidates()
 	// setup a new finalized block which is new enough that satisfies emergency sealing condition
 	for i := 0; i < DefaultEmergencySealingThreshold; i++ {
 		block := unittest.BlockWithParentFixture(ms.LatestFinalizedBlock.Header)
+		ms.ReceiptsDB.On("ByBlockIDAllExecutionReceipts", block.ID()).Return(nil, nil)
```

Comment:
We don't have `ByBlockIDAllExecutionReceipts` anymore, this is out of sync with master or broken

Reply:
thanks. Fixed tests